test/cli/scenarios_to_cli/other /scenario_mutate_endpoint/policy.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion : kyverno.io/v1
+kind : ClusterPolicy
+metadata :
+  name : policy-endpoints
+spec :
```

Comment:
Here and throughout you are adding a space before the colon. While this doesn't cause problems, it's ugly and inconsistent. Please eliminate.